core/common/src/IModel.ts

```diff
@@ -249,6 +252,15 @@ export abstract class IModel implements IModelProps {
     this._ecefTrans = undefined;
   }
 
+  private _geographicCoordinateSystem?: GeographicCRS;
+
+  /** The geographic coordinate reference system of the iModel. */
+  public get geographicCoordinateSystem(): GeographicCRS | undefined { return this._geographicCoordinateSystem; }

```

Comment:
Can you mark this property and `setGeographicCoordinateSystem` as `@alpha` since `GeographicCRS` is marked as such? api-extractor currently does not detect mismatched release tags across packages - I think @calebmshafer filed a bug a while ago.

Reply:
Done